testgrid/pkg/configurator/options/options.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
```

Comment:
I think this should be in the same `configurator` package as well. This options object is only useful with that code, right? (Separate files like this seems good)

It also means that you can manipulate this in other packages as `configurator.Options` instead of `options.Options`

Reply:
No strong preference here. I feel it makes a bit more sense to separate options out, which makes it a little easier to understand the unit test change happened in this PR.

Reply:
@chases2 , please let me know what you think about this

Reply:
I disagree and think it makes more sense to combine the packages, keeping the files separate, but I also don't have a very strong preference.
/lgtm

Reply:
Will keep it as is for now, if it ever become a problem I'll fix